packages/protocol/scripts/foundry/create_and_migrate_anvil_devchain.sh

```diff
@@ -60,6 +60,25 @@ $NON_INTERACTIVE \
 $LIBRARY_FLAGS \
 --rpc-url $ANVIL_RPC_URL || echo "Migration script failed"
 
+# Set the supply for Celo:
+# This supply is just an approximation, but it's importat that this number is non-zero.
+
+# get celo contract address
+CELO_ADDRESS=`cast call $REGISTRY_ADDRESS "getAddressForStringOrDie(string calldata identifier) returns (address)" "CeloToken" --rpc-url $ANVIL_RPC_URL`
+
+# interpersonate the VM, this can't be done from the migration script
+cast rpc anvil_impersonateAccount $CELO_VM_ADDRESS --rpc-url $ANVIL_RPC_URL
+
+# set the balance of the vm address so that it can send a tx
+cast rpc anvil_setBalance $CELO_VM_ADDRESS 1000000000000000000 --rpc-url $ANVIL_RPC_URL
+
+# increase the supply
+# 540001000000000000000000 = (60e3 + 1) * 9 * 10^18
+cast send --from $CELO_VM_ADDRESS --unlocked $CELO_ADDRESS "increaseSupply(uint256)" 540001000000000000000000 --rpc-url $ANVIL_RPC_URL
+
+# stop impersonating the VM address
+cast rpc anvil_stopImpersonatingAccount $CELO_VM_ADDRESS --rpc-url $ANVIL_RPC_URL
```

Comment:
Nit: Adds bash line break `/` to be consistent with how other commands are formatted: 

```suggestion
cast rpc \
anvil_impersonateAccount \
$CELO_VM_ADDRESS \
--rpc-url $ANVIL_RPC_URL

# set the balance of the vm address so that it can send a tx
cast rpc \
anvil_setBalance \
$CELO_VM_ADDRESS 1000000000000000000 \
--rpc-url $ANVIL_RPC_URL

# increase the supply
# 540001000000000000000000 = (60e3 + 1) * 9 * 10^18
cast send \
$CELO_ADDRESS \
"increaseSupply(uint256)" \
540001000000000000000000
--from $CELO_VM_ADDRESS \
--unlocked  \
--rpc-url $ANVIL_RPC_URL

# stop impersonating the VM address
cast rpc \
anvil_stopImpersonatingAccount \
$CELO_VM_ADDRESS \
--rpc-url $ANVIL_RPC_URL
```

See here for example:

https://github.com/celo-org/celo-monorepo/blob/e6dcb576481a9232713b6fb9354a7bdf00e82f33/packages/protocol/scripts/foundry/create_and_migrate_anvil_l2_devchain.sh#L52-L55

Reply:
This is partially done, good enough for me.